src/lib/device-error-handler.js

```diff
@@ -0,0 +1,22 @@
+const deviceControlError = {
```

Comment:
This is just a temporal way to prevent "In control transfer failed" that should change once we handle properly the errors in particle-usb

Reply:
@hugomontero What's preventing us from updating and using these errors from particle-usb directly? Am I missing something here? Example: https://github.com/particle-iot/particle-usb/blob/main/src/error.js and https://github.com/particle-iot/particle-usb/blob/main/src/result.js

Reply:
Yes, there is an story to handle those errors by using the files you mentioned, here I just put the necessary errors and handle it in a better way, since `particle-usb` just return to use `In control transfer failed`
https://app.shortcut.com/particle/story/121834